cmd/install/addon/cmd.go

```diff
@@ -242,8 +268,51 @@ func run(cmd *cobra.Command, argv []string) {
 		})
 	}
 
+	billingModel := args.billingModel
+	billingModelAccountID := args.billingModelAccountID
+
+	if !cmd.Flags().Changed(billingModelFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && !cmd.Flags().Changed(billingModelFlag) {
+		billingModel, err = interactive.GetString(interactive.Input{
```

Comment:
could we use `Interactive.GetOption` here if we have a fixed set of supported options? Otherwise the user will have no idea what options are valid here.